M/msolve/build_tarballs.jl

```diff
@@ -24,6 +24,7 @@ make install
 # platforms are passed in on the command line
 platforms = supported_platforms(; experimental=true)
 filter!(!Sys.iswindows, platforms)  # no FLINT_jll available
+platforms = expand_microarchitectures(platforms)
```

Comment:
Unfortunately this doesn't work well at the moment

Reply:
In which sense? I see the AVX.. builds are successful. Are there problems when the package is loaded in Julia? We have several HPC users who need at least an AVX2 enabled version of msolve_jll in their Julia working environment.

Reply:
Yes, building works fine, but the problem is the artifact selection when loading the package, the logic doesn't work well: you may end up with a slow build or, even worse, with an incompatible one

Reply:
Can you give a rough estimate when we can assume this functionality to be stable? Do you see any other possibility to provide also AVX2 enabled versions in the meantime?

Reply:
Sorry, no timeline at the moment.  And no, I don't see an option for an optimised binary without cutting out older CPUs (apart from the upstream package manually implementing the optimisations and using cpuid instruction to choose at runtime what to do, but I appreciate this isn't a fun path).